src/runtime/slot-proxy.ts

```diff
@@ -67,4 +67,16 @@ export class SlotProxy {
       handler(event);
     }
   }
+
+  /**
+   * gives all rendering capabilities from the current particle to the given one
+   */
+  rewire(particle: Particle): void {
+    const oldParticle = this.particle;
+    this.particle = particle;
+
+    oldParticle.removeSlotProxy(this.slotName);
+    this.particle.addSlotProxy(this);
+    this.particle.renderSlot(this.slotName, ['model', 'template', 'templateName']);
```

Comment:
As I mentioned f2f, there may be some subtlety with the code assuming certain ordering between handle syncing and receiving the slot proxies. I certainly have hit some weirdness in transformation particle (I can explain what these are on our 1:1). I'm ok with you landing this, but just be aware that at some point when we get everything working we will likely come back here and start debugging / attempting to understand and fixing some of these problems.

Still, having some implementation to start with is not bad :)

Reply:
I re-rendered all the slots provided by the reloaded particle and this fixes the problem in tutorial4 for now (where the child particle just disappear when the parent got reloaded).